pkg/epinio/components/application/AppSource.vue

```diff
@@ -416,6 +446,9 @@ export default Vue.extend<Data, any, any, any>({
         />
       </div>
     </template>
+    <template v-else-if="type === APPLICATION_SOURCE_TYPE.GIT_HUB">
+      <GithubPicker @generateUrl="githubUrL" />
```

Comment:
![image](https://user-images.githubusercontent.com/18697775/183471321-ce35e7b0-6800-4a64-9721-26cecba46fd1.png)

The size of the commits table shouldn't be limited to the size of the form fields. it should grow to more of the available space (whilst not making the form field's grow as well)

Reply:
Adding a max to the commit table is a reasonable step, but we shouldn't grow the input fields at the same time which leads to situations like this (width of entire screenshot is 1300px)
![image](https://user-images.githubusercontent.com/18697775/184393268-af9ee7e8-8fe7-4e52-8305-4bb6305eef42.png)